sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -1003,3 +1003,19 @@ INSERT INTO `module_acl_sections` (`section_id`, `section_name`, `parent_section
 SET @group_id = (SELECT `id` FROM `gacl_aro_groups` WHERE `value` = 'admin' LIMIT 1);
 INSERT INTO `module_acl_group_settings` (`module_id`, `group_id`, `section_id`, `allowed`) VALUES (@module_id, @group_id, @section_id+1, 1);
 #EndIf
+
+#IfMissingColumn patient_data name_history
+ALTER TABLE `patient_data` ADD COLUMN `name_history` TINYTEXT;
+#EndIf
+
+#IfMissingColumn patient_data suffix
+ALTER TABLE `patient_data` ADD COLUMN `suffix` TINYTEXT;
+#EndIf
+
+#IfNotRow2D layout_options form_id DEM field_id suffix
+SET @group_id = (SELECT group_id FROM layout_options WHERE field_id='fname' AND form_id='DEM');
+UPDATE `layout_options` SET `seq` = `seq`*10 WHERE group_id = @group_id AND form_id='DEM';
```

Comment:
this is clever :)

Reply:
@sjpadgett , made couple quick changes on this before bringing it in:
- removed the backup group id since lname will always be here and is needed to anchor the suffix
- in database.sql multiplied the seq by 10 there so they stay consistent with the upgrade

Reply:
Okay, thanks @bradymiller